tests/e2e/README.md

```diff
@@ -2,6 +2,20 @@
 
 This directory contains Istio end-to-end tests and associated test framework.
 
+The whole package of E2E tests are not automatically triggered in Before-Merge stage for every PR except a smoke-test (bookinfo test with auth enabled). After-Merge stage is the place to run all E2E tests.
+
+# Running E2E tests in PR stage
+
+E2E-suites can be manually triggered in PR stage with specific command:
+
+`/test e2e-suite-rbac-no_auth`
+
+`/test e2e-suite-rbac-auth`
+
+`/test e2e-cluster_wide-auth`
+
+> NOTE: Please do "/test e2e-*" only after "prow/istio-presubmit" completes, as the e2e tests need the artifacts generated by istio-presubmit.
```

Comment:
So this can be be automatic (i.e. /test waits for presubmit to be done before proceed)?

Reply:
No, it won't. That's why I use your suggestion before.